lib/commands/help/command.js

```diff
@@ -33,20 +32,12 @@ module.exports = class {
   }
 
   getLocalCommandText() {
-    let commands = [
+    const commands = [
       require('../generate/command.json'),
       require('../config/command.json'),
       require('./command.json')
     ];
 
-    let bundler = this.project.model.bundler;
-    if (!bundler || bundler.id === 'cli') {
-      commands = commands.concat([
-        require('../import/command.json'),
```

Comment:
Will this mean that people using the old bundler might miss these commands?

Reply:
`au import` and `au install` had been removed since v1.0.0-beta.1 by the auto-tracer. Since then, they were only print out depreciation warning without doing anything.